remote-config.jsonc

```
@@ -123,6 +123,12 @@
         },
         {
           "message": "Did you know you can set environment variables with the `.ddev/.env` file?"
+        },
+        {
+          "message": "Use `ddev logs --follow --time` for insights on crashed containers or other misbehavior."
+        },
+        {
+          "message": "ddev.com is open-source and the perfect place to share a case study, innovative uses of DDEV, or your custom-made guides and recipes. How-to at https://ddev.com/blog/ddev-website-for-contributors/"
```

I think “open source” should have a space instead of a hyphen here since it’s not used as an adjective. (“custom-made” is the opposite and correct, since it modifies “guides and recipes.”) “How-to” also doesn’t need a hyphen, but it also seems a little vague so I’d recommend being more specific about the call to action:

```suggestion
          "message": "ddev.com is open source and the perfect place to share a case study, innovative uses of DDEV, or your custom-made guides and recipes. Learn how to contribute at https://ddev.com/blog/ddev-website-for-contributors/"
```